three.html

```diff
@@ -0,0 +1,21 @@
+
+    <!DOCTYPE html>
+    <html>
+    <body>
+    
+    
+    <button onclick="openn()">OPen tab</button>
+    <button onclick="closee()">close tab</button>
+
+    <script>
+    function openn() {
+      mywin = window.open("https://www.youtube.com");
+    }
+
+    function closee() {

```

Comment:
Incomplete assignment. Expectations were

`open windows should be close one by one after every 5 seconds and the closing of windows should be stopped on the click stop button.`